### PR TITLE
Added unit test for Polygon and Multipolygon

### LINKED ISSUE #2524: [Code health] Add Unit Test for Polygon and Multipolygon
Add Unit Test for Polygon and Multipolygon

### PR DESCRIPTION
**Pull request by anandwana001**
Fixes #2524 

Added unit test for Polygon and Multipolygon

@sufyanAbbasi @shobhitagarwal1612 PTAL?

Screenshot 2024-06-28 at 13 56 57
Screenshot 2024-06-28 at 13 56 46

### CODE REVIEW

File: ground/src/test/java/com/google/android/ground/model/geometry/GeometryTest.kt [hunk 1]
```diff
@@ -85,13 +85,49 @@ class GeometryTest {
...
+    assertThat(multiPolygon.center())
+      .isEqualTo(Coordinates(lat=-89.62333049749999, lng=41.89413522))
+  }
+
+  @Test
+  fun testMultiPolygonIsEmpty() {
+    val multiPolygon = multiPolygon(polygon(path1, path2), polygon(path3, path4))
+
+    assertThat(multiPolygon.isEmpty()).isEqualTo(false)
+  }
```
Comment (shobhitagarwal1612) [inline ground/src/test/java/com/google/android/ground/model/geometry/GeometryTest.kt:129]: Can we add a test case when isEmpty returns true?

**Review by shobhitagarwal1612 — CHANGES_REQUESTED** [review]

File: ground/src/test/java/com/google/android/ground/model/geometry/GeometryTest.kt (hunk 1 above)
Comment (anandwana001) [reply ground/src/test/java/com/google/android/ground/model/geometry/GeometryTest.kt:129]: yes, we can, but what I checked is it requires a complex params to pass to first make it all polygons non empty.
Will be taking this in next pr.

**Review by shobhitagarwal1612 — APPROVED** [review]

**Pull request merged**